gradoop-examples/src/main/java/org/gradoop/examples/subgraph/SubgraphExample.java

```diff
@@ -73,18 +69,9 @@ public static void main(String[] args) throws Exception {
     // Use these filters with a subgraph call:
     LogicalGraph filteredGraph = inputGraph.subgraph(vertexFilter, edgeFilter);
 
-    // Collect vertices and edges to a local collection to output them both:
-    List<Vertex> resultVertices = new ArrayList<>();
-    List<Edge> resultEdges = new ArrayList<>();
-    filteredGraph.getVertices().output(new LocalCollectionOutputFormat<>(resultVertices));
-    filteredGraph.getEdges().output(new LocalCollectionOutputFormat<>(resultEdges));
-    executionEnvironment.execute();
-
-    // Print results:
-    System.out.println("Vertices:");
-    resultVertices.forEach(System.out::println);
-    System.out.println("Edges:");
-    resultEdges.forEach(System.out::println);
+    // Print result graph
+    filteredGraph.print();
 
+    executionEnvironment.execute();
```

Comment:
This line should be removed, since `LogicalGraph.print()` already calls `execute()`

Reply:
Thx, you are right - done.